.github/workflows/industrial_ci_action.yaml

```diff
@@ -12,10 +12,21 @@ jobs:
       CCACHE_DIR: /github/home/.ccache
     runs-on: ubuntu-latest
```

Comment:
Does `ubuntu-latest` not resolve to `jammy`? I'd have thought you'd need to use `focal`?

Reply:
Same goes for the clang job below.

Reply:
![Screenshot from 2023-11-28 13-23-42](https://github.com/sam-xl/scancontrol/assets/19938667/bec0f1df-b388-4d43-9f35-19cbaecdd52a)
I am surprised too. It seems to have resolved to focal.

Reply:
there is a difference between the image the job is run 'in'/'on' and the environment `industrial_ci` will run your configured jobs in.

Reply:
Ah, okay. Thanks for the clarification!

Since we're not using Github's CI as much as Gitlab right now, maybe we should make an (annotated) example/template like the best practices one we have for Gitlab CI?